cmd/config/generate.go

```diff
@@ -0,0 +1,69 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/chia-network/go-chia-libs/pkg/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v3"
+)
+
+// generateCmd generates a new chia config
+var generateCmd = &cobra.Command{
+	Use:   "generate",
+	Short: "Generate a new chia configuration file",
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg, err := config.LoadDefaultConfig()
+		if err != nil {
+			log.Fatalln(err.Error())
+		}
+
+		err = cfg.FillValuesFromEnvironment()
+		if err != nil {
+			log.Fatalln(err.Error())
+		}
+
+		valuesToSet := viper.GetStringMapString("set")
+		for path, value := range valuesToSet {
+			pathMap := config.ParsePathsFromStrings([]string{path}, false)
+			var key string
+			var pathSlice []string
+			for key, pathSlice = range pathMap {
+				break
+			}
+			err = cfg.SetFieldByPath(pathSlice, value)
+			if err != nil {
+				log.Fatalf("Error setting path `%s` to `%s`: %s\n", key, value, err.Error())
+			}
+		}
+
+		out, err := yaml.Marshal(cfg)
+		if err != nil {
+			fmt.Printf("Error marshalling config: %s\n", err.Error())
```

Comment:
```suggestion
			log.Printf("Error marshalling config: %s\n", err.Error())
```
somewhat random fmt usage, maybe accidental?

Reply:
The import will need to be removed, so if you prefer to do this locally it might be easier.

Reply:
This seems like it should actually be `log.Fatalf` anyway with the os.Exit(1) taken into context.